Agregando Footer de pagina y Font-Awesome (Iconos)

cheperecinos93 opened the pull request:
# Antes de enviar el Pull Request
[X] Confirmo que revisé que mis cambios funcionan
[X ] He adjuntado una imagen de cómo se ven mis cambios (si aplica)
[X] He ejecutado prettier en los archivos de mis cambios

# Intro
-Se ha realizado el componente footer de pagina web, junto con la respectiva adaptabilidad móvil.
-Se ha agregado Font-Awesome para el uso de los iconos de Redes Sociales, y en adelante cualquier otro tipo de icono se necesite.

# Pruebas
Vista en pantalla de Desktop o Laptop
![footer1](https://user-images.githubusercontent.com/15863634/78858488-d2406d80-79e9-11ea-94e1-636da191e7d9.PNG)
Vista móvil
![footer2](https://user-images.githubusercontent.com/15863634/78858482-ce145000-79e9-11ea-880e-13df1fc30d0a.PNG)

cheperecinos93 committed:
Rule para ESLint en caso de errores linebreak-style Unix/Windows

cheperecinos93 committed:
Agregando Font/Awesome - Componente Footer

moudev reviewed (changes requested):
Se necesitan correciones para utilizar las clases de TailwindCSS y nomenclatura de ciertos imports.

moudev committed:
Update linebreak-style eslint rule

moudev committed:
Update classes names and HTML structure of the footer component

Updating HTML structure of the footer component to
use more tailwind classes

  - Updating tailwind css with classes to text and
    background color
  - Adding font-awesome css in the gatsby-browser file
  - Updating HTML structure of the footer component
  - Updating import names in the header component

moudev commented:
Luego de actualización de la estructura HTML y de usar más clases de tailwind.

- **Móvil**

![Footer mobile](https://user-images.githubusercontent.com/13499566/78946901-26e2f780-7a81-11ea-8c08-4941d2a55542.png)


- **Desktop**

![Footer Desktop](https://user-images.githubusercontent.com/13499566/78946916-3104f600-7a81-11ea-8bb1-51426bf2a1d8.png)

cheperecinos93 commented:
Ya los revise los cambios, perfecto

eduarmreyes reviewed (approved):
se ven bien los cambios, ¿ya está listo para hacer merge?
